dspace-api/src/main/java/org/dspace/app/util/ACE.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.util;
+
+import java.util.Set;
+
+import org.apache.log4j.Logger;
+
+/**
+ * Class that represents single Access Control Entry
+ *
+ * @author Michal Josífko
+ * Class is copied from the LINDAT/CLARIAH-CZ (https://github.com/ufal/clarin-dspace) and modified by
+ * @author Milan Majchrak (milan.majchrak at dataquest dot sk)
+ */
+
+public class ACE {
+
+    /** Logger */
+    private static final Logger log = Logger.getLogger(ACE.class);
+    private static final String POLICY_KEYWORD = "policy";
+    private static final String POLICY_DENY_KEYWORD = "deny";
+    private static final String POLICY_ALLOW_KEYWORD = "allow";
+    private static final String ACTION_KEYWORD = "action";
+    private static final String ACTION_READ_KEYWORD = "read";
+    private static final String ACTION_WRITE_KEYWORD = "write";
+    private static final String GRANTEE_TYPE_KEYWORD = "grantee-type";
+    private static final String GRANTEE_TYPE_USER_KEYWORD = "user";
+    private static final String GRANTEE_TYPE_GROUP_KEYWORD = "group";
+    private static final String GRANTEE_ID_KEYWORD = "grantee-id";
+    private static final String ANY_KEYWORD = "*";
+    public static final int ACTION_READ = 1;
+    public static final int ACTION_WRITE = 2;
+    private static final int POLICY_DENY = 1;
+    private static final int POLICY_ALLOW = 2;
+    private static final int GRANTEE_TYPE_USER = 1;
+    private static final int GRANTEE_TYPE_GROUP = 2;
+    private static final String GRANTEE_ID_ANY = "-1";
+    private int policy;
+    private int action;
+    private int granteeType;
+    private String granteeID;
```

Comment:
CONFIG: existing numeric granteeIDs must be replaced with UUID (converted to string)

Reply:
Or the existing numeric granteeIDs could be just converted to a string

Reply:
What I meant was...if the current input-forms.xml have a field with:
```
<acl>
   policy=deny,action=read,grantee-type=user,grantee-id=42
</acl>
```
can that be used as is or do you need to lookup the uuid for eperson 42?

Reply:
Yes already I understand. I did must replace 42 with the eperson's uuid.